scripts/custom/ignore_values.py

```diff
@@ -0,0 +1,19 @@
+from scripts import utils
+
+
+def ignore_values(raw_input):
+    """ Ignore values
+    """
+    if utils.is_empty(raw_input):
+        return None
+
+    values = [
+        "UMLS:C0439673",  # code unknown
+        "C0439673",  # code unknown
+        "UMLS:C1272460",  # code not applicable
+        "C1272460",  # code not applicable
+    ]
+    if raw_input in values:
+        return None
```

Comment:
Return "unknown" instead of None, that way we do not delete the information that was provided in the original table. But instead of doing this with a filter, you could also do this with a concept map ;)